<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ad_deletemenu_Dialog</class>
 <widget class="QDialog" name="ad_deletemenu_Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>330</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>删除菜品</string>
  </property>
  <widget class="QPushButton" name="back_pushButton">
   <property name="geometry">
    <rect>
     <x>-7</x>
     <y>9</y>
     <width>75</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>75</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>105</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(71, 143, 214);</string>
   </property>
   <property name="text">
    <string>编号：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>155</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(71, 143, 214);</string>
   </property>
   <property name="text">
    <string>名称：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>205</y>
     <width>40</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>40</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(71, 143, 214);</string>
   </property>
   <property name="text">
    <string>价格：</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>49</y>
     <width>192</width>
     <height>232</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>20</number>
    </property>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>30</height>
       </size>
      </property>
      <item>
       <property name="text">
        <string>类别</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Cold Dish</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Main Course</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Soup</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="id_lineEdit">
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="name_lineEdit">
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>30</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="price_lineEdit">
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>30</height>
       </size>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="delete_pushButton">
      <property name="minimumSize">
       <size>
        <width>190</width>
        <height>30</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>190</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>删除菜品</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>back_pushButton</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>label_6</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
